homeassistant/components/clarifai/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "clarifai",
+  "name": "Clarifai",
+  "documentation": "https://www.home-assistant.io/integrations/clarifai",
+  "requirements": ["clarifai-grpc==0.1.1"],
+  "dependencies": ["image_processing", "camera"],
```

Comment:
This isn't needed.
```suggestion
```

Reply:
without it, the pre-commit git hook fails:

```
pyupgrade............................................(no files to check)Skipped
black................................................(no files to check)Skipped
codespell............................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
bandit...............................................(no files to check)Skipped
isort................................................(no files to check)Skipped
Check JSON...............................................................Passed
Don't commit to branch...................................................Passed
yamllint.............................................(no files to check)Skipped
mypy.................................................(no files to check)Skipped
gen_requirements_all.....................................................Passed
hassfest.................................................................Failed
- hook id: hassfest
- exit code: 1

Validating json... done in 0.00s
Validating codeowners... done in 0.00s
Validating config_flow... done in 0.08s
Validating dependencies... done in 6.56s
Validating manifest... done in 0.02s
Validating services... done in 0.95s
Validating ssdp... done in 0.00s
Validating translations... done in 0.79s
Validating zeroconf... done in 0.00s
Validating coverage... done in 0.02s

Integrations: 941
Invalid integrations: 1

Found errors. Generating files canceled.

Integration clarifai:
* [DEPENDENCIES] Using component camera but it's not in 'dependencies' or 'after_dependencies'
```

Reply:
Yeah, just depend on camera.